report_compassion/models/contract.py

```diff
@@ -51,7 +51,7 @@ def get_gift_communication(self, product):
             'birthdate': born[lang] + ' ' + birthdate
             if 'Birthday' in product.name else ''
         }
-        return communication.format(**vals)
+        return communication.format(**vals).strip('<br/>')
```

Comment:
Please adapt the unit test in `test_contract.py`, it was expecting the <br/> but now you can remove it to make the tests pass.